rca/static_src/javascript/programmes/components/ProgrammesResults/ProgrammesResults.js

```diff
@@ -0,0 +1,125 @@
+                                Back
+                            </span>
+                        </button>
+                    </div>
+                    <p
+                        className="heading heading--five programmes-results__count"
+                        role="alert"
+                    >
+                        {count}
+                    </p>
+                </div>
+                {programmes.length === 0 ? null : (
+                    <div className="grid">
+                        <div className="programmes-results__list">
+                            {programmes.map((prog) => {
+                                const onHocus = () =>
+                                    setActiveProgramme(prog.id);
+
+                                return (
+                                    <ProgrammeTeaser
+                                        key={prog.id}
+                                        programme={prog}
+                                        onFocus={onHocus}
+                                        onMouseOver={onHocus}
+                                    />
+                                );
+                            })}
+                        </div>
+                        <div className="programmes-results__images">
+                            <div className="programmes-results__images-sticky">
+                                {programmes.map((prog, i) => {
+                                    const isActive =
+                                        activeProgramme === prog.id || i === 0;
+                                    return (
+                                        <img
+                                            key={prog.id}
+                                            className={`programmes-results__image ${
+                                                isActive
+                                                    ? 'programmes-results__image--active'
+                                                    : ''
+                                            }`}
+                                            src={prog.hero_image_square.url}
+                                            width={prog.hero_image_square.width}
+                                            height={
+                                                prog.hero_image_square.height
+                                            }
+                                            alt=""
+                                        />
+                                    );
+                                })}
+                            </div>
+                        </div>
+                    </div>
+                )}
+            </div>
+            <div className="section section--opposite-notch bg bg--dark">
+                <div className="section__notch section__notch--opposite">
+                    <div className="section__notch-fill section__notch-fill--second-col" />
+                </div>
+            </div>
```

Comment:
I can't remember if we added `aria-hidden` to the other notches on the site. If not it's probably out of scope in this PR to do so, but may be worthwhile raising a ticket.

Reply:
👍 I’ll add it here.